From 9f2797d1ab8557b44a1f8d6d4d0b6de368d55e80 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 14 Jan 2022 09:22:59 -0800
Subject: [PATCH] libedit: Include missing header stdc-predef.h

__STDC_ISO_10646__ is defined in stdc-predef.h instead of expecting
every other file including this header, add it here

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 libedit/chartype.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libedit/chartype.h b/libedit/chartype.h
index 319be30..2d3e20c 100644
--- a/libedit/chartype.h
+++ b/libedit/chartype.h
@@ -30,6 +30,9 @@
 #ifndef _h_chartype_f
 #define _h_chartype_f
 
+/*  for __STDC_ISO_10646__ */
+#include <stdc-predef.h>
+
 /* Ideally we should also test the value of the define to see if it
  * supports non-BMP code points without requiring UTF-16, but nothing
  * seems to actually advertise this properly, despite Unicode 3.1 having
-- 
2.34.1

